FormData.js

```diff
@@ -113,7 +116,7 @@ class FormDataPolyfill {
     let map = wm(this)
     name += ''
 
-    return map[name] ? map[name][0] : null
+    return map[name] ? map[name][0][0] : null
```

Comment:
ah, my bad it was never fixed 😆 
We can't simply return the first value in the array since it can contain a blob and a filename. so if you were to do this:
```js
fd.append("f", new Blob, 'filename')
fd.get("f") You would get back just the blob (the value is stored as an array of [value, filename])
```
this is something i fixed in #7